src/snowflake/snowpark/_internal/compiler/telemetry_constants.py

```diff
@@ -6,10 +6,25 @@
 
 
 class CompilationStageTelemetryField(Enum):
+    # types
     TYPE_LARGE_QUERY_BREAKDOWN_OPTIMIZATION_SKIPPED = (
         "snowpark_large_query_breakdown_optimization_skipped"
     )
+    TYPE_COMPILATION_STAGE_STATISTICS = "snowpark_compilation_stage_statistics"
+
+    # keys
     KEY_REASON = "reason"
+    PLAN_UUID = "plan_uuid"
+    TIME_TAKEN_FOR_COMPILATION = "time_taken_for_compilation"
```

Comment:
i believe we the recorded time is second, right? can we put it in the name like time_taken_for_compilation_sec, so that the unit is clear